src/main/java/org/littil/api/mail/MailService.java

```diff
@@ -33,6 +37,16 @@ public void sendContactMail(String recipientEmailAddress, String contactMessage,
         send(template);
     }
 
+    public void sendFeedbackMail(FeedbackPostResource feedback, Optional<String> feedbackEmail) {
```

Comment:
i'd prefer function signature consistency with the other public functions here (use String/simple types), don't use Controller DTO's here.

Reply:
changed to string types